tests/cms/views/poi/test_poi_form.py

```diff
@@ -101,6 +103,39 @@ def create_used_poi(region_slug: str) -> int:
     return used_poi.id
 
 
+def create_unused_poi(region_slug: str, name_add: str = "") -> int:
+    """
+    A helper function to create a new POI that is unused (and therefore deletable)
+    """
+    region = Region.objects.filter(slug=region_slug).first()
+    poi_category = POICategory.objects.first()
+
+    used_poi = POI.objects.create(
```

Comment:
You probably want this to be named `unused_poi`?
```suggestion
    unused_poi = POI.objects.create(
```